autoload/clap/provider/yanks.vim

```diff
@@ -6,13 +6,14 @@ let s:save_cpo = &cpoptions
 set cpoptions&vim
 
 let s:yank_history = []
-let s:yanks = {}
-let s:max_yanks = get(g:, 'clap_provider_yanks_max_entries', 20)
+let s:yanks        = {}
+let s:max_yanks    = get(g:, 'clap_provider_yanks_max_entries', 20)
+let s:min_len      = get(g:, 'clap_provider_yanks_min_len', 1)
 
 function! clap#provider#yanks#collect() abort
   let last_yanked = getreg('"')
 
-  if !empty(s:yank_history) && last_yanked == s:yank_history[0]
+  if !empty(s:yank_history) && last_yanked == s:yank_history[0] || len(last_yanked) < s:min_len
```

Comment:
```suggestion
  if !empty(s:yank_history) && last_yanked == s:yank_history[0] && len(last_yanked) >= s:min_len
```
This is easier to understand IMO.

Reply:
This logic will not work as we need to return if `len(yanked text)` is less than `s:min_len`.
I tested and confirmed.

Reply:
Yeap, the code is not corrent directly, but what I want to express is not to use multiple `** && ** || **` if possible, which makes the priority not so clear in the long term. Then you can try this:

```vim
if len(last_yanked) < s:min_len
  return
endif
```

Reply:
agreed and done.